### DISCUSSION TITLE
Suggestion: generating a release on a regular cadence

### DISCUSSION THREAD

**Discussion by lmilbaum** [dcermak/pytest_container#201] [General]
It would be super helpful if a release is created on a regular cadence. That way, renvoate can pick up the new releases and bump the version used in my project.

**Comment by dcermak** [comment]
I would like to create more frequent releases, but currently the plugin itself sees very little development for even a monthly release cadence. E.g. The first two months of this year had only commits from dependabot.

I will try to create more frequent releases, but I cannot promise any specific cadence, as there is currently really nothing on the roadmap (except for two issues where I'm colossally stuck)

**Reply by lmilbaum** [reply]
Thanks. That makes sense.
